AttitudeManager/FlightModes/Inc/manual.hpp

```diff
@@ -11,6 +11,8 @@ class Manual : public Flightmode {
 
     AttitudeManagerInput run(const AttitudeManagerInput& input) override;
     void updatePid() override;
```

Comment:
Why does manual need pid functions? Is there a planned purpose for these in the near future, or are they redundant but necessary because of inheritance?

Reply:
"Manual" mode may eventually turn into "Acro" mode - I haven't thought about it yet and it's likely they might be split up but...

The PID functions are redundant for now and are necessary because of inheritance (sad), but will likely be used in the future.